src/listeners/substrate-listener/commands/orders/order-fulfilled/order-fulfilled.handler.ts

```diff
@@ -36,24 +36,34 @@ export class OrderFulfilledHandler
 
   async execute(command: OrderFulfilledCommand) {
     await this.logger.log('Order Fulfilled!');
-
     const order = command.orders;
 
     try {
       const isOrderHasBeenInsert =
         await this.loggingService.getLoggingByHashAndStatus(order.id, 3);
-      order.dna_sample_tracking_id = ethers.utils.toUtf8String(
-        order.dna_sample_tracking_id,
-      );
-      order.additional_prices[0].value =
-        Number(order.additional_prices[0].value) / 10 ** 18;
-      order.additional_prices[0].component = ethers.utils.toUtf8String(
-        order.additional_prices[0].component,
-      );
-      order.prices[0].value = Number(order.prices[0].value) / 10 ** 18;
-      order.prices[0].component = ethers.utils.toUtf8String(
-        order.prices[0].component,
-      );
+
+      order.additional_prices[0].value = Number(
+        order.additional_prices[0].value
+          .toString()
+          .split(',')
+          .join('')  
+      ) / 10 ** 18;
+
+      order.prices[0].value = Number(
+        order.prices[0].value
+          .toString()
+          .split(',')
+          .join('')
+      ) / 10 ** 18;
+
+      order.updated_at = new Date(
+        Number(
+          order.updated_at
+            .toString()
+            .split(',')
+            .join('')
+        )
+      )
```

Comment:
This is repeated code, is there a way to make this into a function to be used globally?